R/internals.R

```diff
@@ -197,7 +197,7 @@
     gsub("_Var", "", .) %>%
     gsub("#", "Num_", .) %>%
     gsub("%", "_percent", .) %>%
-    gsub("_Performance", "", .) %>%
+    # gsub("_Performance", "", .) %>%
```

Comment:
for `fb_team_match_log_stats()`, when `stat_type='keeper'`, 'GA' was being returned as a column twice (both for the team and individual). Need to include '_Performance' to distinguish the two